gnmvidispine/vidispine_api.py

```diff
@@ -474,7 +484,7 @@ def raw_request(self,path,method="GET",matrix=None,query=None,body=None,accept="
         if method == "POST" and body is None:
             body = ""
 
-        response=self.sendAuthorized(method,url,body,base_headers)
+        response=self.sendAuthorized(method,url,body,base_headers,rawData=True)
```

Comment:
You should not force rawData=True here, as that will break anything that relies on the string encoding behaviour there. You should make an optional parameter on this method too, and pass it up the callchain.

Reply:
Done.